src/Rings/PBWAlgebra.jl

```diff
@@ -1289,7 +1289,7 @@ julia> eliminate(I, [1, 2 ,3])
 left_ideal(a - 3)
 
 julia> try eliminate(I, [z, a]); catch e; e; end
-ErrorException("no elimination is possible: subalgebra is not admissible")
+
```

Comment:
Why did the error message disappear? Its printing seems to have been part of the doctest on purpose before. Please investigate to make sure not to break a test here.